uportal-war/src/main/java/org/jasig/portal/portlets/portletadmin/PortletAdministrationHelper.java

```diff
@@ -118,13 +112,14 @@
 
 /**
  * Helper methods for the portlet administration workflow.
- * 
+ *
  * @author Jen Bourey, jbourey@unicon.net
  * @revision $Revision$
  */
 @Service
 public class PortletAdministrationHelper implements ServletContextAware {
-	protected final Log logger = LogFactory.getLog(PortletAdministrationHelper.class);
+
+    protected final Log logger = LogFactory.getLog(PortletAdministrationHelper.class);
```

Comment:
If the logger is non-static, it's convenient to use getClass().  (I realize this isn't a line that was added or changed -- other than whitespace -- in this delta.)